meta_recipe/meta.yaml

```diff
@@ -20,9 +20,14 @@ requirements:
     - cudatoolkit {{ cudatoolkit }} #[build_type == 'cuda']
 
 about:
+  home: https://github.com/dmlc/xgboost/
   license: Apache-2.0
+  license_family: APACHE
   license_file: ../LICENSE
```

Comment:
This license_file should probably be removed, since it is the license file for the feedstock.